.github/CODEOWNERS

```diff
@@ -279,19 +279,19 @@
 
 # Interaction project.
 /core/domain/customization_args_util*.py @brianrodri
-/core/domain/expression_parser*.py @aks681
-/core/domain/interaction_registry*.py @aks681
-/core/domain/obj_services*.py @aks681
-/core/templates/components/version-diff-visualization/codemirror-mergeview.directive.ts @aks681
-/core/templates/domain/objects/ @aks681
+/core/domain/expression_parser*.py @iamprayush
+/core/domain/interaction_registry*.py @iamprayush
+/core/domain/obj_services*.py @iamprayush
+/core/templates/components/version-diff-visualization/codemirror-mergeview.directive.ts @iamprayush
```

Comment:
Is this intentional -- this component seems related to the diff / history functionality?

Reply:
You're right. I've moved this to the exploration section now. Thanks!